container-assets/cmd

```diff
@@ -1,3 +1,9 @@
 #!/bin/bash
-update-ca-trust
+
+# Extract to a tmpdir to avoid: p11-kit: couldn't make directory writable: /etc/pki/ca-trust/extracted/pem/directory-hash: Unknown error 1
+mkdir /tmp/extracted
+update-ca-trust extract -o /tmp/extracted/
+rm -rf /etc/pki/ca-trust/extracted/*
+mv /tmp/extracted/* /etc/pki/ca-trust/extracted/
```

Comment:
Do you need to chmod anything after copying it over?

Reply:
I don't think so.  After this, we don't try to write anything in `/etc/pki`.

Reply:
I was more thinking that whatever serves/uses the cert might need the certs to be a particular way (kind of like how things in ~/.ssh need specific perms set)

Reply:
Oh, I'll double check the ownership, but the script appears to set the correct permissions on the directories.  (That's why we're getting the error that prevents us from running it in the first place 😄 )

Reply:
The ownership is `our_uid:root`, but the file and directory permissions are correct.